docker-compose.yml

```diff
@@ -6,53 +6,79 @@ services:
         image: mysql:5.6
         restart: always
         container_name: "akeneo_pim_mysql"
-        env_file: "./.env"
+        environment:
+            - "MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD"
         command: "--max_allowed_packet=1000M --connect_timeout=72"
-        labels:
-            - com.pim.alias.mysql
         volumes:
-            - "./.mysql_data/mysql:/var/lib/mysql"
+            - "mysql_data:/var/lib/mysql"
             - "./conf/mysql/:/etc/mysql/conf.d"
+        ports:
+            - "3306:3306"
         networks:
             akeneo_network:
-                ipv4_address: 192.168.231.20
                 aliases:
                     - mysql.db
 
 
     # Akeneo Pim
     akeneo:
         build: .
-        image: pim/docker-akeneo-apline
+        image: pim/php-fpm-apline
         container_name: "akeneo_pim_app"
+        environment:
+            - "COMPOSER_CACHE_DIR=/cache"
         volumes:
             - "${SOURCE_PATH}:/var/www/pim/"
-            - "./scripts:/var/www/html/scripts"
-            - "./conf:/var/www/html/conf"
-        labels:
-            - com.pim.alias.web_app
+            - "./.composer/cache:/cache"
```

Comment:
I think this makes sense to defer to user config, so that it matches the `$USER_ID` permission because it will involve a write.